webapp/src/marked.tsx

```diff
@@ -385,7 +486,7 @@ export class MarkedContent extends data.Component<MarkedContentProps, MarkedCont
         // Set markdown options
         marked.setOptions({
             renderer: renderer,
-            sanitize: true,
+            sanitize: false,
```

Comment:
Early look, we can't set sanitize to false. This is a security issue with the third party injecting html. Why is this required?

Reply:
Ah - accidentally left that in there. I will set it back to "true".

Reply:
Awesome thank you!